File: AOJ/ITP1/itp_1_10_a.cpp
#include <stdio.h>
#include <cmath>

using namespace std;

int main() {
  double x1, y1, x2, y2;
  scanf("%lf %lf %lf %lf", &x1, &y1, &x2, &y2);
  printf("%.8f\n", sqrt(pow((x2 - x1), 2) + pow((y2 - y1), 2)));
}

File: AOJ/ITP1/itp_1_10_a.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1, y1, x2, y2 float64
	fmt.Scanf("%f %f %f %f", &x1, &y1, &x2, &y2)
	fmt.Println(math.Sqrt(math.Pow(x1-x2, 2) + math.Pow(y1-y2, 2)))
}
